src/functions/views/provider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+use Tribe\Events\Views\V2\Service_Provider as Views;
+
+/**
+ * Checks whether v2 of the Views is enabled or not.
+ *
+ * In order the function will check the `TRIBE_EVENTS_V2_VIEWS` constant,
+ * the `TRIBE_EVENTS_V2_VIEWS` environment variable and, finally, the `static::$option_enabled` option.
+ *
+ * @since TBD
+ *
+ * @return bool Whether v2 of the Views are enabled or not.
+ */
+function tribe_events_views_v2_is_enabled() {
+	if ( defined( 'TRIBE_EVENTS_V2_VIEWS' ) ) {
+		return (bool) TRIBE_EVENTS_V2_VIEWS;
+	}
+
+	$env_var = getenv( 'TRIBE_EVENTS_V2_VIEWS' );
+	if ( false !== $env_var ) {
+		return (bool) $env_var;
+	}
+
+	$enabled = (bool) tribe_get_option( View::$option_enabled, false );
```

Comment:
might want to `use` the `View` class here, getting a fatal if I don't define a constant.